src/entity/application_entity.ts

```diff
@@ -16,13 +16,14 @@ export abstract class ApplicationEntity {
   @PrimaryGeneratedColumn()
   id: number;
 
-  @CreateDateColumn({ type: 'timestamptz', default: 'CURRENT_TIMESTAMP' })
+  // @CreateDateColumn({ type: 'timestamptz', default: 'CURRENT_TIMESTAMP' })
+  @CreateDateColumn()
   createdAt: Date;
 
   @UpdateDateColumn({
-    type: 'timestamptz',
-    default: 'CURRENT_TIMESTAMP',
-    onUpdate: 'CURRENT_TIMESTAMP',
+    // type: 'timestamptz',
+    // default: 'CURRENT_TIMESTAMP',
+    // onUpdate: 'CURRENT_TIMESTAMP',
```

Comment:
```suggestion
    type: 'timestamptz',
    default: 'CURRENT_TIMESTAMP',
    onUpdate: 'CURRENT_TIMESTAMP',
```
sao lại comment chỗ này đi nhỉ, chỗ này anh để là khi nếu object được update thì nó sẽ store lại thời gian cuối cùng được update á em

Reply:
À nãy em xóa vì cú pháp lấy timestamp của mysql khác , e quên chưa sửa lại , để giờ em sửa lại